snafu/run_snafu.py

```diff
@@ -181,35 +182,70 @@ def get_valid_es_document(action, index, index_args):
 
     return es_valid_document
 
-def index_prom_data(prometheus_doc, index_args, action):
+def index_prom_data(index_args, action):
 
     # definition of prometheus data getter, will yield back prom doc
     def get_prometheus_generator(index_args, action):
+        # logger.info(json.dumps(action, indent=4))
```

Comment:
Can we remove the commented out logger (or uncomment it :-) )

Reply:
removed comment